src/libServer/LookupServer.cpp

```diff
@@ -1324,6 +1324,9 @@ Json::Value LookupServer::GetEthTransactionReceipt(const std::string& txnhash) {
 
     std::string blockHash = "";
 
+    std::cout << "Getting TX receipt for: " << txnhash << std::endl;
```

Comment:
Should we log to stdout, or use LOG_GENERAL?

Reply:
This is just debug to assist saeed and me (only a draft pr now for collaboration)